monai/inferers/utils.py

```diff
@@ -44,10 +44,11 @@ def sliding_window_inference(
     sigma_scale: Union[Sequence[float], float] = 0.125,
     padding_mode: Union[PytorchPadMode, str] = PytorchPadMode.CONSTANT,
     cval: float = 0.0,
-    sw_device: Union[torch.device, str, None] = None,
-    device: Union[torch.device, str, None] = None,
+    sw_device: Optional[Union[torch.device, str]] = None,
```

Comment:
`Optional[X]` is equivalent to `Union[X, None]`. https://docs.python.org/3/library/typing.html#typing.Optional so there's no need to update these typing